Misc/NEWS.d/next/Library/2024-08-25-15-24-16.gh-issue-40889.TbsSM1.rst

```diff
@@ -0,0 +1 @@
+Fix `email.utils.parseaddr` to handle multiple hops
```

Comment:
```suggestion
Fix :func:`email.utils.parseaddr` to handle multiple hops.
```

It might be dumb but what's a "hops"?

Reply:
Hops are just the route an email would take specified in the address itself. It's an obsolete artifact of the email specification (https://www.rfc-editor.org/rfc/rfc2822#section-4.4). But it's technically a valid email address where the route part should just be ignored.

Reply:
Oh hops in that sense, ok. I would try to reformulate it because people might be confused about what a "hop" is (AFACIT, this term is not in the RFC). I don't have a better formulation for now though...

Reply:
Sure. Updated the news entry and commit message to be clearer